app/errors/page.js

```diff
@@ -31,9 +33,8 @@ export default async function Errors() {
 
   return (
     <div>
-      {data.error && <Alert message={data.error} />}
```

Comment:
If we take this out, then there's no error message when the api is not available, right?

Reply:
Yeah it looks like that's true (for some other errors it opens a browser alert). It looks there might be ways to implement custom handling of error events in datatables: https://datatables.net/reference/event/dt-error